udata_front/theme/gouvfr/assets/js/components/AdminAddMember/AdminAddMemberModal.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+  <Teleport to="body">
+    <dialog
+      :aria-labelledby="modalTitleId"
+      role="dialog"
+      :id="id"
+      class="fr-modal"
+    >
+      <div class="fr-container fr-container--fluid fr-container-md">
+        <div class="fr-grid-row fr-grid-row--center">
+          <div class="fr-col-12 fr-col-md-8">
+            <div class="fr-modal__body">
+              <div class="fr-modal__header">
+                <button
+                  class="fr-btn--close fr-btn"
+                  :title="t('Close the modal dialog')"
+                  :aria-controls="id"
+                >
+                  {{ t("Close") }}
+                </button>
+              </div>
+              <form
+                @submit.prevent="submit"
+              >
+                <div class="fr-modal__content">
+                  <h1
+                    :id="modalTitleId"
+                    class="fr-modal__title fr-mb-2w"
+                  >
+                    {{ t("Add member to the organization") }}
+                  </h1>
+                  <InputGroup
```

Comment:
We're waiting for the search component here right?

Reply:
Yes ! The API call can be tested by providing an organization id in the input field in the meantime